.gitignore

```diff
@@ -0,0 +1,6 @@
+.terraform
+.terraform.lock.hcl
+terraform.tfstate.backup
+terraform.tfstate
+
+.DS_Store
```

Comment:
please make at the end of one code document a empty line . That make for that not a problem but older programming language have problem without an empty line at the end. I write this comment here but that is for all other code documents

Reply:
done 👍